packages/browser/src/tracing/request.ts

```diff
@@ -113,10 +117,43 @@ export function instrumentOutgoingRequests(_options?: Partial<RequestInstrumenta
   const shouldAttachHeadersWithTargets = (url: string): boolean => shouldAttachHeaders(url, tracePropagationTargets);
 
   const spans: Record<string, Span> = {};
+  const client = getClient();
 
   if (traceFetch) {
+    if (client) {
+      // Keeping track of http requests, whose body payloads resolved later than the intial resolved request (e.g. SSE)
+      client.addEventProcessor(event => {
+        if (event.type === 'transaction' && event.spans) {
+          event.spans.forEach(span => {
+            if (span.op === 'http.client') {
+              const updatedTimestamp = spanIdToEndTimestamp.get(span.span_id);
+              if (updatedTimestamp !== undefined) {
```

Comment:
```suggestion
              if (updatedTimestamp) {
```

I think we can do this, `0` is not a value we care about here so this saves a few bytes :)